test/e2e/bundle_e2e_test.go

```diff
@@ -0,0 +1,111 @@
+package e2e
+
+import (
+	"context"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
+)
+
+var _ = Describe("Installing bundles with new object types", func() {
+	var (
+		kubeClient     operatorclient.ClientInterface
+		operatorClient versioned.Interface
+		dynamicClient  dynamic.Interface
+	)
+
+	BeforeEach(func() {
+		kubeClient = ctx.Ctx().KubeClient()
+		operatorClient = ctx.Ctx().OperatorClient()
+		dynamicClient = ctx.Ctx().DynamicClient()
+	})
+
+	AfterEach(func() {
+		cleaner.NotifyTestComplete(true)
+	})
+
+	When("a bundle with a pdb, priorityclass, and VPA object is installed", func() {
+		By("including the VPA CRD in the CSV")
+		const (
+			packageName = "busybox"
+			channelName = "alpha"
+			subName     = "test-subscription"
+		)
+
+		BeforeEach(func() {
+			const (
+				sourceName = "test-catalog"
+				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
+			)
+			// create catalog source
+			source := &v1alpha1.CatalogSource{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       v1alpha1.CatalogSourceKind,
+					APIVersion: v1alpha1.CatalogSourceCRDAPIVersion,
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      sourceName,
+					Namespace: testNamespace,
```

Comment:
Sorry, what's the value of this `testNamespace` variable?

Reply:
it depends on whether its run locally or on CI
* locally its `operators` 
* on openshift CI its `openshift-operators`